streams/mx.m3u

```diff
@@ -127,8 +127,6 @@ https://5ca39be538307.streamlock.net/telemetrika/smil:telemetrika.smil/playlist.
 https://video1.getstreamhosting.com:1936/8172/8172/playlist.m3u8
 #EXTINF:-1 tvg-id="RCGTV2.mx",RCG TV 2 (360p) [Not 24/7]
 https://5caf24a595d94.streamlock.net:1937/stream56/stream56/playlist.m3u8
-#EXTINF:-1 tvg-id="",Romanza+ Africa (720p)
-https://origin3.afxp.telemedia.co.za/PremiumFree/romanza/playlist.m3u8
```

Comment:
works totally fine for me. Does it need a removal?
![image](https://github.com/iptv-org/iptv/assets/5430116/16d6e4b8-60a2-482f-a183-0734e7748527)

Reply:
[I moved it here.](https://github.com/iptv-org/iptv/pull/14997/files#diff-c89720c3c1bb8c4172c3f0da66547995d2092bd7b0f12dc400edaa9b18da60b0R50-R51)

Reply:
belongs to TV Azteca Networks ([TV Azteca](https://en.wikipedia.org/wiki/TV_Azteca)) from Mexico

Reply:
Done moved it back